Configuration/Skimming/python/PDWG_EXODisappTrk_cff.py

```diff
@@ -11,8 +11,21 @@
    throw = False,
    andOr = True,
    HLTPaths = [
-      #"HLT_MET105_IsoTrk50_v*",
-      "MC_PFMET_v17"
+      "HLT_PFMET*_PFMHT*_IDTight_v*",
+      "HLT_PFMETTypeOne*_PFMHT*_IDTight_v*",
+      "HLT_PFMETNoMu*_PFMHTNoMu*_IDTight_v*", 
+      "HLT_MET*_IsoTrk*_v*", 
+      "HLT_PFMET*_*Cleaned_v*", 
+      "HLT_Ele*_WPTight_Gsf_v*", 
+      "HLT_Ele*_WPLoose_Gsf_v*", 
+      "HLT_IsoMu*_v*", 
+      "HLT_IsoMu*_TightChargedIsoPFTauHPS*_Trk1_eta2p1_SingleL1_v*", 
+      "HLT_IsoMu*_eta2p1_TightChargedIsoPFTauHPS*_eta2p1_CrossL1_v*",
+      "HLT_IsoMu*_MediumChargedIsoPFTauHPS20_Trk1_eta2p1_SingleL1_v*", 
```

Comment:
Just noting that the first pattern is already a superset of the last 3 (and includes more than those, e.g. `HLT_IsoMu50_AK8PFJet230_SoftDropMass40_v`).

Reply:
this has been fixed